docs/bigquery.md

```diff
@@ -1,5 +1,29 @@
 # BigQuery
 
+## Generating tables
+
+A worker in Google Cloud has a [crontab](https://github.com/HTTPArchive/bigquery/blob/master/crontab) that regularly runs scripts to generate BigQuery tables based on the HTTP Archive data.
+
+### HAR
+
+HAR tables are generated by running `sync_har.sh` with the client type (chrome or android) and the optional test date:
+
+`sync_har.sh chrome 2017-05-15`
+
+This depends on the bucket in Google Storage. If the bucket does not contain a `done` file, the sync is aborted. The file may be set days after the crawl completes.
+
+The script is scheduled to run daily for chrome and android.
+
+### Requests and Pages
+
+The pages and requests tables are generated by running `sync_csv.sh` with the date and an optional `mobile_` prefix:
```

Comment:
Would it make sense to merge "generating tables" and this section? They're part of the same pipeline, right?